agenda/data/src/main/java/com/gkp/agenda/data/OfflineAgendaRepository.kt

```diff
@@ -24,6 +29,8 @@ class OfflineAgendaRepository(
     private val scope: CoroutineScope,
     private val syncAgendaItems: SyncAgendaItems,
     private val updatedAgendaItemsDao: UpdatedAgendaItemsDao,
+    private val deletedAgendaItemsDao: DeletedAgendaItemsDao,
+    private val createdAgendaItemsDao: CreatedAgendaItemsDao
```

Comment:
I always try to be as clear with the intentions behind my code as possible. Choosing the right name for classes and functions can be hard. That is why I prefer more descriptive names above short names. 

Here, I would add the phrase `pendingSync` to these DAOs to communicate that the entities in these tables need to be sync.

```
PendingSyncUpdateAgendaItemsDao,
PendingSyncDeleteAgendaItemsDao,
PendingSyncCreateAgendaItemsDao
```

Again, just my personal preference 😄